src/components/timeline/TrackBandwidth.css

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.timelineTrackBandwidthGraph {
+  position: relative;
+  width: 100%;
+  height: var(--graph-height);
+}
+
+.timelineTrackBandwidthCanvas {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+.timelineTrackBandwidthGraphDot {
+  position: absolute;
+  width: 6px;
+  height: 6px;
+  border-radius: 3px;
+  margin-top: -3px;
+  margin-left: -3px;
+  background-color: var(--blue-60);
```

Comment:
We have a `TRACK_BANDWIDTH_COLOR` but also specify the dot color here with a hardcoded color. Would it make sense to merge them maybe? We also have a `getDotColor` function.

Reply:
The Power and CustomMarker tracks use the getDotColor function, the memory track specifies its dot color in the css file. I think the reason for using the getDotColor function for power and custom marker tracks was that the color can be user specified for these tracks, whereas for the memory (and bandwidth) tracks it never changes.

I can change this code to use `getDotColor(TRACK_BANDWIDTH_COLOR)` to have the bandwidth color specified only in a single place. Would you like me to change the memory track too to have all of them be consistent?

Reply:
Ah I see. It's good to keep them consistent indeed. Thanks for changing them